profile/README.md

```diff
@@ -3,19 +3,19 @@
 TWIN
 </h1>
 
-Welcome to the TWIN repository, to learn more about the TWIN foundation visit [https://www.twin.org/](https://www.twin.org/).
+Welcome to the TWIN repository. To learn more about the TWIN foundation, visit [https://www.twin.org/](https://www.twin.org/).
 
 Documentation for development using TWIN can be found at [https://twindev.org/](https://twindev.org/)
 
 To install the code for all the framework repositories, refer to the [Workspace repository](https://github.com/twinfoundation/workspace) which includes all the other repositories as submodules. Follow the instructions in the repository to learn how to use it.
 
 ## Applications
 
-### Workbench Server [[Repo](https://github.com/twinfoundation/workbench)]
+### Workbench Server [[Repo's app folder](https://github.com/twinfoundation/workbench/tree/next/apps/workbench-server)]
```

Comment:
These links should point to the `main` branch not `next`

Reply:
Good point, fixed

Reply:
@obany we broke the links because on `master` there is only the `api-server`...

https://github.com/twinfoundation/workbench/tree/main/apps/api-server

Should we release the lastest to master? or point to `next`?